src/cdk/v2/destinations/rakuten/data/propertiesMapping.json

```diff
@@ -1,41 +1,45 @@
 [
   {
-    "sourceKeys": "properties.orderId",
+    "sourceKeys": ["properties.orderId", "properties.order_id"],
```

Comment:
We should prefer the fields from standard spec.

Reply:
Same for other fields as well.

Reply:
done